.github/workflows/ccpp.yml

```diff
@@ -85,7 +85,16 @@ jobs:
     steps:
       - name: Install dependencies
         run: |
-          sudo apt-get install catch2 cmake g++ libboost-dev libboost-filesystem-dev libboost-timer-dev libhdf5-mpi-dev libparmetis-dev libpugixml-dev mpi-default-dev ninja-build pkg-config
+          sudo apt-get install catch2 cmake g++ libboost-dev libboost-filesystem-dev libboost-timer-dev libgflags-dev libhdf5-mpi-dev libparmetis-dev libpugixml-dev mpi-default-dev ninja-build pkg-config
+          git clone https://github.com/google/glog.git
+          cd glog
+          cmake -S . -B build -G "Unix Makefiles"
+          cmake --build build
+          sudo cmake --build build --target install
+          cd ..
+          sudo rm -rf glog
```

Comment:
Cant we use: https://packages.ubuntu.com/search?keywords=google-glog here, i.e.
https://packages.ubuntu.com/noble/libgoogle-glog-dev

Reply:
Comments on the glog switch on Chris's PR. I'm just doing some incremental merges as this cannot go in until glog is done (loguru is not working on Windows).

Reply:
I'm trying out spdlog now, I think it is better.